src/components/GCChart/GCChart.tsx

```diff
@@ -113,7 +113,7 @@ function GCChart({ title, config, data, yFormatter }: GCChartParams) {
                 key={`area-${nextKey()}`}
                 type="monotone"
                 dataKey={area.key}
-                stroke="#00a6d1"
+                stroke="#19c0ea"
```

Comment:
```suggestion
                stroke={colors.blue}
```

Reply:
The colours in that component are hardcoded placeholders put there by Romanas. I think better tidy these all up in one go.